test/test_helper.rb

```diff
@@ -14,6 +14,7 @@
 WebMock.disable_net_connect!(
   allow_localhost: true,
   allow: [
+    'core.dev.collectionspace.org',
```

Comment:
Will allow for this version of the csv importer but don't want to replicate in future.